recipes/libcurl/all/conanfile.py

```diff
@@ -126,19 +127,21 @@ def source(self):
         tools.download("https://curl.haxx.se/ca/cacert.pem", "cacert.pem", verify=True)
 
     def build(self):
-        self._patch_misc_files()
-        if self.settings.compiler != "Visual Studio":
-            self._build_with_autotools()
-        else:
-            self._build_with_cmake()
+        with tools.chdir(self.source_folder):
```

Comment:
why do you need chdir now if it was working before? You can pass `configure_folder` to AutotoolsBuildEnvironment.configure() if you need to customize.

Reply:
configure_folder is not exactly the same as being in the directory, so it would have different behaviour when using the development flow and create command

Reply:
removed the chdir action